pkg/descheduler/strategy_migration.go

```diff
@@ -151,4 +152,67 @@ var pluginsMap = map[string]func(ctx context.Context, nodes []*v1.Node, params *
 			klog.V(1).ErrorS(err, "plugin finished with error", "pluginName", removeduplicates.PluginName)
 		}
 	},
+	"HighNodeUtilization": func(ctx context.Context, nodes []*v1.Node, params *api.StrategyParameters, handle *handleImpl) {
+		var (
+			thresholds       = make(componentconfig.ResourceThresholds)
+			targetThresholds = make(componentconfig.ResourceThresholds)
+		)
+		for key, value := range params.NodeResourceUtilizationThresholds.Thresholds {
+			thresholds[key] = componentconfig.Percentage(value)
+		}
+		args := &componentconfig.HighNodeUtilizationArgs{
+			ThresholdPriority:          *params.ThresholdPriority,
+			ThresholdPriorityClassName: params.ThresholdPriorityClassName,
+			Thresholds:                 thresholds,
+			TargetThresholds:           targetThresholds,
+			UseDeviationThresholds:     params.NodeResourceUtilizationThresholds.UseDeviationThresholds,
+			NumberOfNodes:              params.NodeResourceUtilizationThresholds.NumberOfNodes,
+		}
+
+		if err := validation.ValidateHighNodeUtilizationArgs(args); err != nil {
+			klog.V(1).ErrorS(err, "unable to validate plugin arguments", "pluginName", nodeutilization.HighNodeUtilizationPluginName)
+			return
+		}
+		pg, err := nodeutilization.NewHighNodeUtilization(args, handle)
+		if err != nil {
+			klog.V(1).ErrorS(err, "unable to initialize a plugin", "pluginName", nodeutilization.HighNodeUtilizationPluginName)
+			return
+		}
+		status := pg.(framework.BalancePlugin).Balance(ctx, nodes)
+		if status != nil && status.Err != nil {
+			klog.V(1).ErrorS(err, "plugin finished with error", "pluginName", nodeutilization.HighNodeUtilizationPluginName)
+		}
+	},
+	"LowNodeUtilization": func(ctx context.Context, nodes []*v1.Node, params *api.StrategyParameters, handle *handleImpl) {
+		var (
+			thresholds       = make(componentconfig.ResourceThresholds)
+			targetThresholds = make(componentconfig.ResourceThresholds)
+		)
+		for key, value := range params.NodeResourceUtilizationThresholds.Thresholds {
+			thresholds[key] = componentconfig.Percentage(value)
+		}
+
+		args := &componentconfig.LowNodeUtilizationArgs{
+			ThresholdPriority:          *params.ThresholdPriority,
+			ThresholdPriorityClassName: params.ThresholdPriorityClassName,
+			Thresholds:                 thresholds,
```

Comment:
Passing `params.NodeResourceUtilizationThresholds.Thresholds` and `params.NodeResourceUtilizationThresholds.TargetThresholds` directly instead of `thresholds`/`targetThresholds` did not work?